cns/hnsclient/hnsclient_windows.go

```diff
@@ -53,6 +53,9 @@ const (
 	// Name of the loopback adapter needed to create Host NC apipa network
 	hostNCLoopbackAdapterName = "LoopbackAdapterHostNCConnectivity"
 
+	// Name of the loopback adapter needed to search for Host NC apipa network
+	vEthernethostNCLoopbackAdapterName = "vEthernet (LoopbackAdapterHostNCConnectivity)"
```

Comment:
this has the `hostNCLoopbackAdapterName` substring, should the matcher be `contains` instead of strict equals?

Reply:
@rbtr The API that it will end up calling to is net.InterfaceByName(iFaceName) and that will do eaxct match so that's the reason we got the Apipa loopback interface at first place.

Reply:
can `vEthernethostNCLoopbackAdapterName` be "vEthernet (" + hostNCLoopbackAdapterName + ")"?